packs/equipment/rhino-shot.json

```diff
@@ -36,7 +36,33 @@
             "title": "Pathfinder Lost Omens: The Grand Bazaar"
         },
         "quantity": 1,
-        "rules": [],
+        "rules": [
+            {
+                "damageType": "force",
+                "diceNumber": 2,
+                "dieSize": "d6",
+                "key": "DamageDice",
+                "selector": "{item|_id}-damage"
+            },
+            {
+                "key": "Note",
+                "outcome": [
+                    "success"
+                ],
+                "selector": "{item|_id}-damage",
+                "text": "PF2E.SpecificRule.Equipment.RhinoShot.Success",
+                "title": "{item|name}"
```

Comment:
We'll now need a localized label for this, otherwise it will show the weapon name.